### DISCUSSION TITLE
詳解 4 種類型的爬蟲技術

### DISCUSSION THREAD

**Discussion by leon10456** [1101-GS4719-C/Python-Community-GS4719-C#559] [General]
大家學py應該都會想要學一些爬蟲幫自己做事對吧
以下是常見的四種爬蟲類型介紹
https://www.readfog.com/a/1630495861465255936

**Comment by ThomasMeng0515** [comment]
Python爬蟲的教學影片YouTube上也有很多，也滿容易學的，感謝分享~

**Comment by lx3211** [comment]
爬商品資訊的時候，假設商品搜索結果有很多頁，要注意request的時間間隔，避免DoS人家。

**Comment by HaoFish123** [comment]
感謝分享讓我對爬蟲技術有初步的認識。
